app/src/org/commcare/dalvik/activities/HomeButtons.java

```diff
@@ -136,6 +141,7 @@ public void onClick(View v) {
     private static View.OnClickListener getIncompleteButtonListener(final CommCareHomeActivity activity) {
         return new View.OnClickListener() {
             public void onClick(View v) {
+                reportButtonClick(GoogleAnalyticsFields.LABEL_INCOMPLETE_FORMS_BUTTON);
```

Comment:
Seems a little redundant to log an event here and also within the `FormRecordListActivity` that is launched by pressing this button.